src/mame/heathkit/z37_fdc.cpp

```diff
@@ -226,14 +226,14 @@ void heath_z37_fdc_device::device_add_mconfig(machine_config &config)
 
 void heath_z37_fdc_device::set_irq(u8 data)
 {
-	LOGLINES("set irq, allowed: %d data: %d\n", m_intrq_allowed, data);
+	LOGLINES("set irq, allowed: %d data: %d\n", m_irq_allowed, data);
 
-	m_fd_irq_cb(m_intrq_allowed ? data : CLEAR_LINE);
+	m_irq_cb(m_irq_allowed ? data : 0);
 }
```

Comment:
I just noticed now, but this is attached to a write line callback on the FDC chip – it works because devcb instantiates adapters automatically, but conventionally we would use `set_irq(int state)` since it’s a single line rather than multiple lines in the bits.  Same applies for `set_drq`.